src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck: do work");
+
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck: stop work");

```

Comment:
Similarly, the message in the `stopWork` method should be more informative and indicate that a 'Truck' has stopped working.